include/cartogram_info.hpp

```diff
@@ -37,7 +37,7 @@ class CartogramInfo
   void read_geojson(const std::string &, bool, std::string &);
   std::map<std::string, InsetState> &ref_to_inset_states();
   void replace_missing_and_zero_target_areas();
-  void set_map_name(const std::string &);
+  std::string set_map_name(const std::string &);
```

Comment:
A function ideally should have only one task. Maybe we can move the "getter" function out of this function to a separate function.

Reply:
Apparently, conventions are changing!
https://stackoverflow.com/questions/1345001/is-it-bad-practice-to-make-a-setter-return-this

I feel it is better to return the value in this case, as the setter does modify the value.

Reply:
I've expanded on the comment above to make it even clearer.